design-documents/asynchronous-import/modularity/data-converting.md

```diff
@@ -0,0 +1,146 @@
+# Data converting before import
```

Comment:
I also would like to recommend that please don't implement your own ETL code, there are many good PHP libraries available that can do all this for you. Please have a look into those

https://php-etl.readthedocs.io/en/v1/
https://blog.panoply.io/6-of-the-best-php-etl-tools

Reply:
Thanks for info, I will check it!

Reply:
@tariqjawed83, I expect that its will take a lot of time to confirm from Architecture team of usage some library in Magento Core?